webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -113,12 +114,12 @@ const HostDetails = ({ match, location: { hash } }) => {
             {/* TODO: extracting text and remove timeago usage in favor i18n */}
             {response.name ? (
               <div>
-                created{' '}
+                {__('created')}{' '}
                 <RelativeDateTime
                   date={response.created_at}
                   defaultValue="N/A"
                 />{' '}
-                by {response.owner_name} (updated{' '}
+                {sprintf(__('by %s (updated'), response.owner_name)}{' '}
                 <RelativeDateTime
                   date={response.updated_at}
                   defaultValue="N/A"
```

Comment:
I don't think the string should be exported like this.
It should be exported like whole, or not at all, now we are exporting just parts...

exporting partial strings for translation is not the best practice.
This need the new intl (#7910), which allows usage of components within text.

Reply:
should we wrap the whole thing and pass it with a div in `dangerouslySetInnerHtml`?

Reply:
Can we just disable the eslint plugin for this chunk of code?
It should be changed after we upgrade jed and intl to new versions.

Reply:
reverted to the original usage